rails-questions.md

```diff
@@ -2,33 +2,34 @@
 
 1. What is the command to create a new Rails application with a Postgres database?
 
-
+    rails new application_name -d postgresql
 
 2. Explain each section of the following route:  get '/library/:id' => 'book#show'
 
-get -
-library -
-:id -
-book -
-show -
-
+get - The HTTP verb being sent to the server via the URL
+library - The first part of the URL
+:id - The parameter being passed through the URL
+book - Name of the controller
+show - Name of the method in the controller
 
 3. Oops, I forgot to add a foreign key to my model. Describe the steps to remedy this mistake.
 
-
-
+    Create a migration. In the migration file, add a column to the belongs to table with the modelname_id of the model you want it to be associated to. Complete the migration. 
```

Comment:
Pretty good, but I would consider it asking for the steps, and format your answer like so. Pretend you are having to take a newbie to rails through the entire process. 

E.g.
"Step 1: Generate a new migration file by running command `rails g migration add_foreign_key_to_model`
Step 2: Open the new migration file found in db/migrations.." 
etc.